hrdf/a-2084-01-parallel-gripper.hrdf

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<robot version="1.1.0">
```

Comment:
End effectors are only supported in v1.2.0, so we will want to update this version number (same with other files where you added a gripper)

Reply:
I've incremented the gripper files to 1.2.0, left the others at 1.1.0 for now.